src/ViewLayer/DebugDisplay/BatteryPage/ProgressBar.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <QWidget>
+#include <QPaintEvent>
+#include <QPainter>
+#include <QProgressBar>
+
+class ProgressBar : public QWidget
+{
+    Q_OBJECT
+public:
+    ProgressBar(QWidget* parent = 0);
+    ~ProgressBar();
+    double progress;
```

Comment:
I'm not a fan of how this is a public variable, this should be a private variable and have a setter